client/signup/steps/plans/index.jsx

```diff
@@ -138,6 +181,17 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		if ( showPlanFeatures ) {
+			return (
+				/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
@mtias You're welcome to jump in on the branch with the whitespace fixes. I don't think it's really worth the time to fix these up, since devs are highly unlikely to modify the SVGs directly. We'd copy and paste an exported file into these methods instead.

As an aside, I'm not a huge fan of having these SVG's inlined in the component, as they're rather distracting. Right now we have a copy of the SVGs being served in the public folder, and while it would be a separate request, max-age is set appropriately so they'd be cached while you're unproxied. Are we basically worried about the first page load on a cold cache?